sfa_dash/api_interface/__init__.py

```diff
@@ -17,7 +19,13 @@ def get_request(path, **kwargs):
     requests.Response
         The api response.
     """
-    return requests.get(f'{app.config["SFA_API_URL"]}{path}', **kwargs)
+    try:
+        session = auth0  # already has token ready to use for api
+    except Exception:
+        # make a requests session?
+        # redirect to login?
+        raise
+    return session.get(f'{app.config["SFA_API_URL"]}{path}', **kwargs)
```

Comment:
I think that we don't necessarily need to make this try/except here if all the endpoints besides / are protected, we can probably safely assume that the user at least has a session.